person.rb

```diff
@@ -24,7 +24,14 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  public :can_use_services?
+
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] I am not able to create a `rental` please address that
![rent](https://user-images.githubusercontent.com/44745855/187952083-8697baac-9f49-458c-8418-ce421c2302f2.png)